app/cli.py

```diff
@@ -155,6 +157,29 @@ def apikey():
         """apikey commands"""
         pass
 
+    @app.cli.command()
+    def check_bad_url():
```

Comment:
This name can be improved. The name indicates you are only checking one url instead of a mass action.

Reply:
fixed